gravitee/values.yaml

```diff
@@ -49,7 +53,7 @@ api:
   replicaCount: 1
   image:
     repository: graviteeio/management-api
-    tag: 1.15.0
+    tag: 1.19.3
```

Comment:
it would be good to document which versions are supported by this chart

Reply:
I will add details on the supported version.